sonar-docs/analysis/scan/sonarscanner-for-azure-devops.md

```diff
@@ -52,7 +52,7 @@ Each extension provides three tasks you will use in your build definitions to an
 * **Run Code Analysis** task, to actually execute the analysis of the source code. 
    * This task is not required for Maven or Gradle projects, because scanner will be run as part of the Maven/Gradle build.
 * **Publish Quality Gate Result** task, to display the Quality Gate status in the build summary and give you a sense of whether the application is ready for production "quality-wise". 
-   * This tasks is optional. 
+   * This tasks is mandatory if you are using a release pipeline, with the {instance} quality gate status check pre-deployment gate, otherwise it's optional. 
```

Comment:
Pre-existing typo: `This task is mandatory...` (_task_ should be singular)

Reply:
Thanks, this has been modified.